src/igvfd/tests/data/inserts/curated_set.json

```diff
@@ -57,9 +51,6 @@
         "documents": [
             "/documents/igvf:characterization_FACS_insert/"
         ],
-        "publication_identifiers": [
```

Comment:
Would you be able to add "publications" into some of the inserts where "publication_identifiers" was removed? (Or other objects that didn't previously have publication_identifiers") Just want to make sure we have some examples of links to "publications".

Reply:
I added them in the prior ticket which added the property `publications`. But I can add more here.

Reply:
I see. In that case probably don't need to add much more, I didn't realize it was already in some inserts.